### PR TITLE
Point enterprise metrics templates at registry.access.redhat.com/openshift3/met…

### PR DESCRIPTION
**Pull request by sdodson**
…rics-

@mwringe can you check that this is right?

### CODE REVIEW

**Comment by sdodson** [comment]
In particular should we be pinning it to 3.1.0 or latest ?

**Comment by mwringe** [comment]
3.1.0 should pull in the latest build for 3.1.0

'latest' might be a problem in the future if we have build specific for 3.2 or 4.0 or whatever and someone is wanting to run the version for 3.1.0

**Comment by sdodson** [comment]
@mwringe Ok, so +1 on this change as is?

**Comment by mwringe** [comment]
I ran out of time to test it today and I will have to check tomorrow. From a quick glance it looks right.

File: roles/openshift_examples/files/examples/v1.0/infrastructure-templates/enterprise/metrics-deployer.yaml [hunk 1]
```diff
@@ -81,11 +81,11 @@ parameters:
 -
   description: 'Specify prefix for metrics components; e.g. for "openshift/origin-metrics-deployer:v1.1", set prefix "openshift/origin-"'
   name: IMAGE_PREFIX
-  value: "hawkular/"
+  value: "registry.access.redhat.com/openshift3/metrics-"
```
Comment (mwringe) [inline roles/openshift_examples/files/examples/v1.0/infrastructure-templates/enterprise/metrics-deployer.yaml]: This needs to be "registry.access.redhat.com/openshift3/"

File: roles/openshift_examples/files/examples/v1.1/infrastructure-templates/enterprise/metrics-deployer.yaml [hunk 2]
```diff
@@ -81,11 +81,11 @@ parameters:
 -
   description: 'Specify prefix for metrics components; e.g. for "openshift/origin-metrics-deployer:v1.1", set prefix "openshift/origin-"'
   name: IMAGE_PREFIX
-  value: "hawkular/"
+  value: "registry.access.redhat.com/openshift3/metrics-"
```
Comment (mwringe) [inline roles/openshift_examples/files/examples/v1.1/infrastructure-templates/enterprise/metrics-deployer.yaml]: This needs to be "registry.access.redhat.com/openshift3/"

**Comment by mwringe** [comment]
Other than the change needed to the IMAGE_PREFIX value, it seems to be working for me.

File: roles/openshift_examples/files/examples/v1.0/infrastructure-templates/enterprise/metrics-deployer.yaml (hunk 1 above)
Comment (sdodson) [inline roles/openshift_examples/files/examples/v1.0/infrastructure-templates/enterprise/metrics-deployer.yaml]: ok, updated

File: roles/openshift_examples/files/examples/v1.1/infrastructure-templates/enterprise/metrics-deployer.yaml (hunk 2 above)
Comment (sdodson) [inline roles/openshift_examples/files/examples/v1.1/infrastructure-templates/enterprise/metrics-deployer.yaml]: ok, updated

**Comment by sdodson** [comment]
@brenton fixes enterprise metrics templates to use enterprise default image naming

**Pull request merged**